pip_test.go

```diff
@@ -71,7 +69,6 @@ func testPipInstall(t *testing.T, isLegacy bool) {
 				test.args = append([]string{"pip", "install"}, test.args...)
 			}
 			testPipCmd(t, createPipProject(t, test.outputFolder, test.project), strconv.Itoa(buildNumber), test.moduleId, test.expectedDependencies, test.args)
-
```

Comment:
Let's re-add this new-line, to remain consistent with the existing structure of this function.